examples/kotlin-matter-controller/java/src/com/matter/controller/commands/common/IPAddress.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ *   Copyright (c) 2023 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+package com.matter.controller.commands.common
+
+import java.net.InetAddress
+
+class IPAddress(var address: InetAddress) {
+  fun getHostAddress(): String {
```

Comment:
why do we have this helper? it would replace `foo.address.hostAddress` with `foo.getHostAddress` ... that is one line with another one-line. Does not seem worth it.